src/scripts/towns/TownList.ts

```diff
@@ -4708,12 +4708,77 @@ const LumioseDexio = new NPC('Dexio', [
+], {
+    image: 'assets/images/temporaryBattle/Calem.png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 1), new QuestLineStepCompletedRequirement('A Beautiful World', 3, GameConstants.AchievementOption.less)]),
+});
+
+const TeamFlareGrunt1 = new NPC('Team Flare Grunt', [
+    'What\'s this? Well, well. What do we have here? A nosy little trainer has come poking around.',
+    'Listen up! We\'re the fashionable team whose very name makes people tremble in fear: Team Flare!',
+    'Team Flare\'s goal is to make it so we\'re the only ones who are happy! We don\'t care one bit about what happens to other trainers or Pokémon.',
+    'Get out of here, kid. Don\'t you know not to play with fire? We\'ll obliterate you!',
+], {
+    image: 'assets/images/trainers/Team Flare Grunt (male).png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 3), new QuestLineStepCompletedRequirement('A Beautiful World', 5, GameConstants.AchievementOption.less)]),
+});
+
+const FossilScientist = new NPC('Fossil Scientist', [
+    'Why, hello! Here to look for fossils as well?',
+    'What did you say? Teemphlair was here? What\'s that? A Pokémon?',
+    'You are very lucky! I just now found a fossil! But it\'s one I already have, so I\'ll give it to you!',
+    'Well, I\'m going back to the lab, goodbye!',
+], {
+    image: 'assets/images/trainers/Scientist (male).png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 5), new QuestLineStepCompletedRequirement('A Beautiful World', 7, GameConstants.AchievementOption.less)]),
+});
+
+const TeamFlareGrunt2 = new NPC('Team Flare Grunt', [
+    'Stop right there! I remember you! You\'re the one who foiled our fossil-finding plans!',
+    'And now you\'re here, standing around these stones. Do you have any idea what they even are?',
+    'Of course you don\'t! You know nothing about the legend of 3000 years ago that says...',
+    'Oh, whatever! Who cares about all of that? It\'s time for me to get some sweet revenge... with style!',
+], {
+    image: 'assets/images/trainers/Team Flare Grunt (male).png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 7), new QuestLineStepCompletedRequirement('A Beautiful World', 9, GameConstants.AchievementOption.less)]),
+});
+
+const TeamFlareGrunt3 = new NPC('Team Flare Grunt', [
+    'Hear this, pip-squeak! Team Flare is gonna use the stones on Route 10 and a fantastic treasure to make everyone on our team happy!',
+    'In this world the ones who have power are the ones who win with flair!',
+
+], {
+    image: 'assets/images/trainers/Team Flare Grunt (male).png',
+    requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 9), new QuestLineStepCompletedRequirement('A Beautiful World', 10, GameConstants.AchievementOption.less)]),
```

Comment:
The disappear requirement here should be set to 11, so it doesn't disappear instantly upon being read

Reply:
This one is on purpose. It's supposed to disappear instantly after the conversation and you cannot "follow" him.